airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeS3StagingSqlOperations.java

```diff
@@ -64,12 +64,8 @@ public String uploadRecordsToStage(final JdbcDatabase database,
                                      final SerializableBuffer recordsData,
                                      final String schemaName,
                                      final String stageName,
-                                     final String stagingPath)
-      throws Exception {
-    AirbyteSentry.executeWithTracing("UploadRecordsToStage",
```

Comment:
why are you removing the sentry tracing?

Reply:
Since we are going to move to Datadog, I thought that Sentry no longer matters. But I will add it back in the interim.